### PR TITLE
Run test database setup/cleaning commands synchronously

### PR DESCRIPTION
**Pull request by mlaszloffy**
The database setup commands were running alongside the test execution. Reorganized flyway commands so that copying of data is not needed. Docker updates to better cleanup and ensure we're working with a clean db.

Jira ticket:

- [ ] Includes a change file
- [ ] Updates developer documentation

### CODE REVIEW

File: test/signoffTest.js [hunk 1]
```diff
@@ -63,8 +63,9 @@ const getAllSignoffs = (server) => {
 };
 
 describe('case sign-off tracking', () => {
-  beforeEach(async () => {
-    await cmd.run('npm run fw:test-clean; npm run fw:test-migrate');
+  before(function () {
+    this.timeout(5000);
```
Comment (grapigeau) [inline test/signoffTest.js:67]: Is this one being half the wait time deliberate?

**Review by grapigeau — APPROVED** [review]

File: test/setup_test_dbs.sh [hunk 2]
```diff
@@ -1,36 +1,30 @@
...
 export SQLITE_LOCATION=$(pwd)
-sqlite3 fpr.db < migrations/create_fpr_table.sql
+sqlite3 fpr.db < create_fpr_table.sql
 echo "Recreated FPR test table"
 cd "${CURRENT}"
 
-docker run --rm -v $(pwd)/test:/flyway/conf -v $(pwd)/test/sql:/flyway/sql --network=host flyway/flyway clean && \
-		docker run --rm -v $(pwd)/test:/flyway/conf -v $(pwd)/test/sql:/flyway/sql --network=host flyway/flyway migrate
+# wait for PG_DB_CONTAINER_NAME to be ready
+while ! docker exec -it pgtestdb pg_isready -U postgres; do sleep 1; done
```
Comment (grapigeau) [inline test/setup_test_dbs.sh:27]: I think using `-it` broke the github testing see:
`the input device is not a TTY`

Comment (mlaszloffy) [reply test/setup_test_dbs.sh]: removed, thanks!

**Review by grapigeau — APPROVED** [review]

File: test/archiveTest.js [hunk 3]
```diff
@@ -55,9 +55,10 @@ const isValidDate = (date) => {
 };
 
 describe('case archive tracking', () => {
-  beforeEach(async () => {
-    await cmd.run('npm run fw:test-clean; npm run fw:test-migrate');
-  });
+   before(function () {
+     this.timeout(10000);
+     cmd.runSync('npm run fw:test-clean; npm run fw:test-migrate');
+   });
```
[the remark this answers is not in the document]
Comment (mlaszloffy) [reply test/archiveTest.js:61]: It appears that these tests are independent of each other. They will be grouped differently or beforeEach used if that changes.

**Review by djcooke — APPROVED** [review]

**Pull request merged**